packages/graphql-types-generator/src/loading.ts

```diff
@@ -16,7 +16,12 @@ function loadIntrospectionSchema(schemaPath: string): GraphQLSchema {
   if (!fs.existsSync(schemaPath)) {
     throw new ToolError(`Cannot find GraphQL schema file: ${schemaPath}`);
   }
-  const schemaData = require(schemaPath);
+  let schemaData;
+  try {
+    schemaData = require(schemaPath);
+  } catch {
+    schemaData = JSON.parse(fs.readFileSync(schemaPath, 'utf8'));
```

Comment:
mock-fs does not effect `require`. `require(schemaPath);` and `JSON.parse(fs.readFileSync(schemaPath, 'utf8'));` are otherwise equivalent in this use case. When we move re-merge the graphql-generator `loadIntrospectionSchema` should not be executed anymore.